placekey/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,72 @@
+## Reference
+
+### PLACEKEY
+
+#### VERSION
+
+{{% bannerNote type="code" %}}
+placekey.VERSION()
+{{%/ bannerNote %}}
+
+Returns the current version of the placekey library.
+
+#### PLACEKEY_FROMH3
```

Comment:
I'm not sure about this `FROMH3` and `H3_FROMPLACEKEY` reinventing the wheel and changing how we work with h3 indexes (using string instead of int64 and so on).